### PR TITLE
changed the file paths of the parsed and preprocessed data to the path data/name.of.file

### PR DESCRIPTION
**Pull request by piEsposito**
changed the file paths of the parsed and preprocessed data to the path data/name.of.file

it was as dataname.of file, so I added the "/". if the preprocessed data is not on data/ folder, train.py does not run

did the same on glove files, so the NN can be trained with train.py script

### CODE REVIEW

**Comment by piEsposito** [comment]
just made the changes requested. thank you!

File: examples/reading_comprehension/prepare_data.py [hunk 1]
```diff
@@ -20,9 +20,11 @@
 import json
 import os
 from tqdm import tqdm
+import os
```
Comment (peteriz) [inline examples/reading_comprehension/prepare_data.py]: double import

Comment (piEsposito) [reply examples/reading_comprehension/prepare_data.py]: fixed the double import

**Comment by piEsposito** [comment]
fixed the double import

**Pull request merged**